packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -178,19 +174,36 @@ export const TableRows = ({
                 );
               }
 
-              if (hasReviewWorkflows && name === 'strapi_reviewWorkflows_stage') {
-                return (
-                  <Td key={key}>
-                    {data.strapi_reviewWorkflows_stage ? (
-                      <ReviewWorkflowsStage
-                        color={data.strapi_reviewWorkflows_stage.color}
-                        name={data.strapi_reviewWorkflows_stage.name}
-                      />
-                    ) : (
-                      <Typography textColor="neutral800">-</Typography>
-                    )}
-                  </Td>
-                );
+              if (hasReviewWorkflows) {
+                switch (name) {
+                  case 'strapi_reviewWorkflows_stage':
+                    return (
+                      <Td key={key}>
+                        {data.strapi_reviewWorkflows_stage ? (
+                          <ReviewWorkflowsStage.ReviewWorkflowsStageEE
+                            color={data.strapi_reviewWorkflows_stage.color}
+                            name={data.strapi_reviewWorkflows_stage.name}
+                          />
+                        ) : (
+                          <Typography textColor="neutral800">-</Typography>
+                        )}
+                      </Td>
+                    );
+                  case 'strapi_assignee':
+                    return (
+                      <Td key={key}>
+                        {data.strapi_assignee ? (
+                          <ReviewWorkflowsStage.ReviewWorkflowsAssigneeEE
+                            firstname={data.strapi_assignee.firstname}
+                            lastname={data.strapi_assignee.lastname}
+                          />
+                        ) : (
+                          <Typography textColor="neutral800">-</Typography>
+                        )}
+                      </Td>
+                    );
+                  default:
```

Comment:
Should we make this if/ else, since `default` doesn't really make sense here and we don't want to add a new column through the feature anytime soon?